tutorials/operators/gradient_framework.ipynb

```diff
@@ -0,0 +1,1427 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 70,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Gradient  [(1.4142135623730954+1.12e-16j), 0j]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Assign the parameters and evaluate the gradient\n",
+    "grad_result = grad.assign_parameters(value_dict).eval()\n",
+    "print('Gradient ', grad_result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Gradients w.r.t. State Parameters\n",
+    "A gradient w.r.t. a state parameter may be evaluated with different methods. Each method has advantages and disadvantages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 71,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ComposedOp([\n",
+      "  OperatorMeasurement(SummedOp([\n",
+      "    0.5 * X,\n",
+      "    -1.0 * Z\n",
+      "  ])),\n",
+      "  CircuitStateFn(\n",
+      "             ┌───┐┌───────┐┌───────┐\n",
+      "  q156778_0: ┤ H ├┤ RZ(a) ├┤ RX(b) ├\n",
+      "             └───┘└───────┘└───────┘\n",
+      "  )\n",
+      "])\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Define the Hamiltonian with fixed coefficients\n",
+    "H = 0.5 * X - 1 * Z\n",
+    "# Define the parameters w.r.t. we want to compute the gradients\n",
+    "params = [a, b]\n",
+    "# Define the values to be assigned to the parameters\n",
+    "value_dict = { a: np.pi / 4, b: np.pi}\n",
+    "\n",
+    "# Combine the Hamiltonian observable and the state into an expectation value operator\n",
+    "op = ~StateFn(H) @ CircuitStateFn(primitive=qc, coeff=1.)\n",
+    "print(op)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Parameter Shift Gradients\n",
+    "<a id='param_shift_grad'></a>\n",
+    "Given a Hermitian operator $g$ with two unique eigenvalues $\\pm r$ which acts as generator for a parameterized quantum gate $$G(\\theta)= e^{-i\\theta g}.$$\n",
+    "Then, quantum gradients can be computed by using eigenvalue $r$ dependent shifts to parameters. \n",
+    "All [standard, parameterized Qiskit gates](https://github.com/Qiskit/qiskit-terra/tree/master/qiskit/circuit/library/standard_gates) can be shifted with $\\pi/2$, i.e.,\n",
+    " $$ \\frac{\\partial\\langle\\psi\\left(\\theta\\right)|\\hat{O}\\left(\\omega\\right)|\\psi\\left(\\theta\\right)\\rangle}{\\partial\\theta} = 2 \\left(\\langle\\psi\\left(\\theta+\\pi/2\\right)|\\hat{O}\\left(\\omega\\right)|\\psi\\left(\\theta+\\pi/2\\right)\\rangle -\\partial\\langle\\psi\\left(\\theta-\\pi/2\\right)|\\hat{O}\\left(\\omega\\right)|\\psi\\left(\\theta-\\pi/2\\right)\\rangle\\right).$$\n",
```

Comment:
Is rhs of param shift rule`\frac{...}{...} / 2` instead of `2 \frac{...} {...}` ?

Reply:
You are right